rust/relay/ebpf-turn-router/src/main.rs

```
@@ -380,6 +395,40 @@ fn try_handle_ipv6_channel_data_to_udp(
     Ok(())
 }
 
+#[inline(always)]
+fn learn_interface_ipv4_address(ipv4: &Ip4) -> Result<(), Error> {
+    let interface_addr = match INT_ADDR_V4.get_ptr_mut(0) {
+        Some(addr) => addr,
+        None => return Err(Error::InterfaceIpv4AddressAccessFailed),
```

This only returns `None` when using out-of-bounds access. We could try to panic here and see if the optimizer can optimize that away? Otherwise we could at least make the error message a bit clearer.

---

I don't think we're allowed to panic right? If that got optimized away this would just be a silent failure I think.

I can update the error log.